openwisp_notifications/handlers.py

```diff
@@ -25,7 +25,6 @@
 logger = logging.getLogger(__name__)
 
 EXTRA_DATA = app_settings.get_config()['USE_JSONFIELD']
-EMAIL_BATCH_INTERVAL = app_settings.EMAIL_BATCH_INTERVAL
```

Comment:
This is executed at project startup, during tests it won't be possible to override it (trying to override it would have no effect, because this line was already computed before the override).